client/my-sites/current-site/notice.jsx

```diff
@@ -80,25 +81,21 @@ export class SiteNotice extends React.Component {
 			return null;
 		}
 
-		const eventName = 'calypso_domain_credit_reminder_impression';
 		const eventProperties = { cta_name: 'current_site_domain_notice' };
 		const { translate } = this.props;
 
 		return (
-			<Notice
-				isCompact
-				status="is-success"
-				icon="info-outline"
-				text={ translate( 'Free domain available' ) }
-			>
-				<NoticeAction
-					onClick={ this.props.clickClaimDomainNotice }
-					href={ `/domains/add/${ this.props.site.slug }` }
-				>
-					{ translate( 'Claim' ) }
-					<TrackComponentView eventName={ eventName } eventProperties={ eventProperties } />
-				</NoticeAction>
-			</Notice>
+			<UpsellNudge
```

Comment:
Should we include this under the `upsell/nudge-component` feature flag added in https://github.com/Automattic/wp-calypso/pull/38810?

Reply:
Whoops, that shouldn't have made it into this. 😅 That's been my test case, and yes, it needs to be under a feature flag for A/B testing. Will remove that changeset!